.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__
+test_GAS_assign.py
```

Comment:
Can you remove these @sgsutcliffe from `.gitignore` (the `test_*.py` entries)? I'm wondering if they would mess up with tracking changes of our tests